include/nanogui/common.h

```diff
@@ -109,39 +111,47 @@ struct GLFWcursor;
 #endif // DOXYGEN_SHOULD_SKIP_THIS
 
 // Define command key for windows/mac/linux
-#ifdef __APPLE__
-/// If on OSX, maps to ``GLFW_MOD_SUPER``.  Otherwise, maps to ``GLFW_MOD_CONTROL``.
-#define SYSTEM_COMMAND_MOD GLFW_MOD_SUPER
+#if defined(__APPLE__) || defined(DOXYGEN_DOCUMENTATION_BUILD)
+    /// If on OSX, maps to ``GLFW_MOD_SUPER``.  Otherwise, maps to ``GLFW_MOD_CONTROL``.
+    #define SYSTEM_COMMAND_MOD GLFW_MOD_SUPER
 #else
-/// If on OSX, maps to ``GLFW_MOD_SUPER``.  Otherwise, maps to ``GLFW_MOD_CONTROL``.
-#define SYSTEM_COMMAND_MOD GLFW_MOD_CONTROL
+    #define SYSTEM_COMMAND_MOD GLFW_MOD_CONTROL
 #endif
 
 NAMESPACE_BEGIN(nanogui)
 
-/// Cursor shapes available to use in GLFW.
+/// Cursor shapes available to use in GLFW.  Shape of actual cursor determined by Operating System.
 enum class Cursor {
-    Arrow = 0,
-    IBeam,
-    Crosshair,
-    Hand,
-    HResize,
-    VResize,
-    /// Not a cursor --- should always be last: enables a loop over the cursor types.
-    CursorCount
+    Arrow = 0,  ///< The arrow cursor.
+    IBeam,      ///< The I-beam cursor.
+    Crosshair,  ///< The crosshair cursor.
+    Hand,       ///< The hand cursor.
+    HResize,    ///< The horizontal resize cursor.
+    VResize,    ///< The vertical resize cursor.
+    CursorCount ///< Not a cursor --- should always be last: enables a loop over the cursor types.
 };
 
 /* Import some common Eigen types */
-using Eigen::Vector2f;
-using Eigen::Vector3f;
-using Eigen::Vector4f;
-using Eigen::Vector2i;
-using Eigen::Vector3i;
-using Eigen::Vector4i;
-using Eigen::Matrix3f;
-using Eigen::Matrix4f;
-using Eigen::VectorXf;
-using Eigen::MatrixXf;
+/// Type alias to allow ``Eigen::Vector2f`` to be used as ``nanogui::Vector2f``.
+typedef Eigen::Vector2f Vector2f;
```

Comment:
Breathe chokes on raw `using` in a namespace (it does seem to do ok with them in a class).  Without explicit typedefs here, the fact that `nanogui::Vector3f` etc are available never show up in the docs.

This can easily be undone, at the expense of no explicit documentation of the vector types.

Reply:
Typedefs are so C++03 :(. Can we at least do ``using VectorXf = Eigen::VectorXf``, etc?

Reply:
I'll look into `using` tomorrow, I think I tried that and it broke but don't remember.  I know `using x = thing` is ok in classes though.

(sorry if you get a lot of emails from edits, the github review thing is nice but i overcommented the first one.......)